src/test/java/org/cornutum/regexpgen/js/GenerateTest.java

```diff
@@ -491,6 +510,54 @@ private void verifyNotMatchesNone( String regexp)
       is( Optional.empty()));
     }
 
+  private void verifyMatchesForSpaceChars( String regexp, Set<Character> spaceChars)
```

Comment:
For simplicity and better test coverage, should use the String form of `spaceChars`.